modules/getweblinks.py

```diff
@@ -84,6 +86,15 @@ def add_red(link):
     colors = Bcolors()
     return '\t' + colors.On_Red + link + colors.ENDC
 
+def new_func(link):
```

Comment:
Rename the function to possibly `def get_link_status(link):`?

Reply:
done